openmls/src/group/mls_group/proposal.rs

```diff
@@ -384,4 +386,58 @@ impl MlsGroup {
 
         Ok((mls_message, proposal_ref))
     }
+
+    /// Updates Group Context Extensions
+    ///
+    /// Commits to the Group Context Extension inline proposal using the [`Extensions`]
+    ///
+    /// Returns an error when the group does not support all the required capabilities
+    /// in the new `extensions` or if there is a pending commit.
+    /// // FIXME: #1217
+    #[allow(clippy::type_complexity)]
```

Comment:
Let's not make this part of the public doc comment.

```suggestion
    //// FIXME: #1217
```